packages/node/test/global-setup.jest.ts

```diff
@@ -0,0 +1,63 @@
+import TicTacToeApp from "@counterfactual/apps/build/TicTacToeApp.json";
+import MinimumViableMultisig from "@counterfactual/contracts/build/MinimumViableMultisig.json";
+import ProxyFactory from "@counterfactual/contracts/build/ProxyFactory.json";
+import dotenvExtended from "dotenv-extended";
+import { ContractFactory, Wallet } from "ethers";
+import { Web3Provider } from "ethers/providers";
+import fs from "fs";
+import ganache from "ganache-core";
+import mkdirp from "mkdirp";
+import os from "os";
+import path from "path";
+
+dotenvExtended.load();
```

Comment:
Isn't the `--setupFiles dotenv-extended/config` thing supposed do this?

Reply:
It apparently doesn't work on setup files.

I assumed it did on my first try but the values ended up being `undefined`.